tests/src/test/scala/whisk/core/controller/test/WebActionsApiTests.scala

```diff
@@ -699,20 +699,36 @@ trait WebActionsApiTests extends ControllerTestCommon with BeforeAndAfterEach wi
         it should s"handle http web action and provide defaults (auth? ${creds.isDefined})" in {
             implicit val tid = transid()
 
+            def confirmEmptyResponse() = {
+                status should be(NoContent)
+                response.entity shouldBe HttpEntity.Empty
+                withClue(headers) {
+                    headers.length shouldBe 0
+                }
+            }
+
             Seq(s"$systemId/proxy/export_c.http").
                 foreach { path =>
-                    allowedMethods.foreach { m =>
-                        invocationsAllowed += 1
-                        actionResult = Some(JsObject())
+                    Set(JsObject(), JsObject("body" -> "".toJson), JsObject("body" -> JsNull)).
+                        foreach { bodyResult =>
+                            allowedMethods.foreach { m =>
+                                invocationsAllowed += 2
+                                actionResult = Some(bodyResult)
+
+                                m(s"$testRoutePath/$path") ~> Route.seal(routes(creds)) ~> check {
+                                    withClue(s"failed for: $bodyResult") {
+                                        confirmEmptyResponse()
+                                    }
+                                }
 
-                        m(s"$testRoutePath/$path") ~> Route.seal(routes(creds)) ~> check {
-                            status should be(OK)
-                            response.entity shouldBe HttpEntity.Empty
-                            withClue(headers) {
-                                headers.length shouldBe 0
+                                // repeat with accept header, which should be ignored for content-negotiation
+                                m(s"$testRoutePath/$path") ~> addHeader("Accept", "application/json") ~> Route.seal(routes(creds)) ~> check {
```

Comment:
Looks like we established a pattern of using ```MediaTypes.`application/json`.value``` instead of just using `application/json`. Six other of the same references below.

Reply:
I deliberately use the string when setting custom headers, but the constant when checking the expected result. I rather leave it this way since it mimics the actual header values one might set literally.